src/apps/main/electron/deeplink/process-login.test.ts

```diff
@@ -20,12 +19,9 @@ describe('process-login', () => {
     // Given
     refreshMock.mockResolvedValue({ data: { user: { uuid: 'uuid' } } });
     // When
-    await processLogin({ search: '?mnemonic=bW5lbW9uaWM=&newToken=bmV3VG9rZW4=&privateKey=cHJpdmF0ZUtleQ==' });
+    const promise = processLogin({ search: '?mnemonic=bW5lbW9uaWM=&newToken=bmV3VG9rZW4=&privateKey=cHJpdmF0ZUtleQ==' });
     // Then
-    call(loggerMock.error).toStrictEqual({
-      msg: 'Cannot process login deeplink',
-      error: new Error('Invalid mnemonic: mnemonic'),
-    });
+    await expect(promise).rejects.toThrowError('Invalid mnemonic: mnemonic');
```

Comment:
Are you sure you want to only assert the message and not the error instance itself ?